std_msgs/Header header
int32 hole_id
string strategy
float32 stepdown
float32 feedrate
geometry_msgs/PoseStamped[] poses
